configs/recognition/slowonly/slowonly_r50_video_4x16x1_256e_kinetics400_rgb.py

```diff
@@ -93,7 +93,7 @@
         pipeline=test_pipeline))
 # optimizer
 optimizer = dict(
-    type='SGD', lr=0.1, momentum=0.9,
+    type='SGD', lr=0.3, momentum=0.9,
```

Comment:
double check that this lr is for 8 gpus

Reply:
Yes, that is for 8 gpu (since we use batch size 24 on each GPU